doc/nrf/device_guides/working_with_nrf/nrf54l/ug_nrf54l15_gs.rst

```diff
@@ -56,299 +54,13 @@ Complete the following steps to test if the PDK works correctly:
 
 If something does not work as expected, contact Nordic Semiconductor support.
 
-.. _nrf54l15_gs_installing_software:
+.. _ug_nrf54l15_gs_installing_software:
```

Comment:
There is a sentence, in this file,
"In the limited sampling release, the nRF54L15 PDK comes preprogrammed with the Blinky sample."
The sentence "In the limited sampling release" can likely be removed here.

Reply:
In the Next steps chapter, there are links to some PDFs that requires authentication.

Reply:
removed the line. 
The PDFs are leftovers from the cs3 release, I think we need to discuss what to do with them and if they're still applicable